libcrux-sha3/src/simd/avx2.rs

```diff
@@ -241,4 +241,8 @@ impl KeccakItem<4> for Vec256 {
     fn split_at_mut_n(a: [&mut [u8]; 4], mid: usize) -> ([&mut [u8]; 4], [&mut [u8]; 4]) {
         split_at_mut_4(a, mid)
     }
+
+    fn store<const RATE: usize>(_state: &[[Self; 5]; 5], _out: [&mut [u8]; 4]) {
```

Comment:
These todos need to be implemented in this round right?

Reply:
I don't know if we have a need for an x2 or x4 incremental API? In ML-DSA we only need the incremental API when taking in the message. In any case I've added an issue to track this for later: #482